src/test/util/txmempool.cpp

```diff
@@ -82,6 +82,19 @@ std::optional<std::string> CheckPackageMempoolAcceptResult(const Package& txns,
                                 wtxid.ToString());
         }
 
+        if (!atmp_result.m_replaced_transactions.empty() && mempool) {
+            LOCK(mempool->cs);
+            // If replacements occurred and it used 2 transactions, this is a package RBF and should result in a cluster of size 2
+            if (atmp_result.m_wtxids_fee_calculations.has_value() && atmp_result.m_wtxids_fee_calculations.value().size() == 2) {
+                for (const auto& fee_calc_wtxid : atmp_result.m_wtxids_fee_calculations.value()) {
+                    const auto& entry = *Assert(mempool->GetEntry(fee_calc_wtxid));
+                    if (entry.GetCountWithDescendants() + entry.GetCountWithAncestors() != 3) {
+                        return strprintf("tx %s has too many ancestors or descendants for a package rbf", fee_calc_wtxid.ToString());
+                    }
```

Comment:
91a7e8ba7606b68401b29d2609c240a604ab6fe1: this isn't a perfect test for cluster size 2... why not call `CheckConflictTopology`?

Reply:
Directly checked cluster size instead.

Reply:
Nice, agree that's better. However e4e14fedc9622ca7cfc40af4f2aa70ed4bb7daa6 is now unnecessary and can be dropped.

Reply:
even better. dropped.